pyflakes/checker.py

```diff
@@ -1471,6 +1480,34 @@ def SUBSCRIPT(self, node):
                 self.handleChildren(node)
             finally:
                 self._in_typing_literal = orig
+        elif (
+                (
+                    isinstance(node.value, ast.Name) and
+                    node.value.id == 'Annotated'
+                ) or (
+                    isinstance(node.value, ast.Attribute) and
+                    node.value.attr == 'Annotated'
+                )
+        ):
+            orig = self._in_typing_annotated
```

Comment:
instead of a new attribute, I think you can visit the annotation part with `_in_annotation` True and visit the non-annotation part with it `False`

Reply:
Done